lib/src/nav-tabs/Tab.tsx

```diff
@@ -149,11 +137,31 @@ const Tab = styled.a<{
 
 const TabIconContainer = styled.div<{ iconPosition: NavTabsPropsType["iconPosition"] }>`
   display: flex;
+  font-size: 24px;
+  color {
+    ${(props) => props.theme.disabledIconColor};
+  }
+  &[aria-selected="true"] {
```

Comment:
Instead of using the aria pass the selected prop to be used also in the styled component.

Reply:
Chenged!

Reply:
color styling can be removed from svg and  &[aria-selected="true"] since we have it already applied in the parent element.

Reply:
the props added to the declaration of the styled component also need to be added above when the styled component is used.